packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -268,6 +268,64 @@ const prepareY2Axis = (scale, tickValues, height, format) => d3
   .tickValues(tickValues)
   .tickFormat(format);
 
+/**
+ * Creates the Y and Y2 axis format when tick values are in between -0.5 to 0.5 or -0.05 to 0.05 or -0.005 to 0.005 and so on.
+ *
+ * @private
+ * @param {Array} tickValues - Array of values that represent the tick values
+ * @param {Object} axisData - Object which consists of Y or Y2 axis data.
+ * @returns {object} d3 object which forms the y2-axis scale
+ */
+const getYAndY2AxisFormat = (tickValues, axisData) => {
+  let tickDecimalMagnitude = 0;
+  let largestDecimalValue = 0;
+  tickValues.forEach((tick) => {
+    if (tick !== 0) {
+      let tempTick = tick;
+      if (tick < 0) {
+        tempTick = -tick;
```

Comment:
nit: We could condense these lines by using [Math.abs()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs).

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/243/commits/27703ff8837d602d95699fbac3a81b2ffbd9e2cc#diff-9df3d9f47180537037ee4dbff72c1e27837425023bf945c6c96135d788e54298R285).